src/combinatorics/simple_graph/triangle/basic.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import combinatorics.simple_graph.clique
+
+/-!
+# Triangles in graphs
+
+This file defines triangles in simple graphs.
+
+A triangle is a `3`-clique, that is a set of three vertices which are all connected.
```

Comment:
```suggestion
A triangle is a `3`-clique, which is a set of three vertices that are all adjacent.
```

Reply:
That's not the same sentence!

Reply:
The original sentence is both grammatically incorrect and incorrect though. (What does it mean for the vertices to be connected? That there's a path between them? That their induced graph is connected? If it's that they're all adjacent, then might as well say that.)

There's an important distinction between "that" and "which". Maybe this describes the rule: "that" adds a predicate to restrict, "which" reminds us about a predicate. This is just reminding us about what a `3`-clique is, right? so use "which", but then "that" is used to add an additional property that the three vertices must hold.

Reply:
Does this make more sense with regards to my intention?
```suggestion
A triangle is a `3`-clique. That is, a set of three vertices which are all adjacent.
```

Reply:
This looks fine to me, but I would just use "... , i.e., ...".

Reply:
It's still grammatically incorrect, and now the sentence has no subject. If you *really* don't want to use my correction, here are some options to choose from:
```
A triangle is a `3`-clique. That is, it is a set of three vertices that are all adjacent.
A triangle is a `3`-clique, that is, a set of three vertices that are all adjacent.
```